package.json

```diff
@@ -4,7 +4,8 @@
     "jest": "^29.6.2"
   },
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "test:traffic-light": "jest test_frameworks/trafficLight.test.js"
   },
   "type": "module"
 }
```

Comment:
package.json file should have a basic structure. PLease look into documentation:
https://nodesource.com/blog/the-basics-of-package-json